### PR TITLE
[zh]update content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

### PR DESCRIPTION
**Pull request by zyy19981018**

### CODE REVIEW

**Comment by tengqm** [comment]
dup of #33598.

@zyy19981018 please check previous comments to your PRs and join the team on bigger efforts.

File: content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 1]
```diff
@@ -380,11 +340,77 @@ If you no longer have this command, or the token has expired, you can run `kubea
...
 
+{{% /tab %}}
+
+{{% tab name="Docker Engine" %}}
+
+<!--
+#### Install Docker Engine
+Install the `Containers` feature
+-->
+#### 安装 Docker 引擎
```
Comment (javadoors) [inline content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:351]:
```suggestion
#### 安装 Docker 引擎
安装 `Containers` 特性。
```

File: content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 2]
```diff
@@ -380,11 +340,77 @@ If you no longer have this command, or the token has expired, you can run `kubea
...
+
+<!--
+Install Docker
+Instructions to do so are available at [Install Docker Engine - Enterprise on Windows Servers](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server#install-docker).
+
+[Install cri-dockerd](https://github.com/Mirantis/cri-dockerd) which is required so that the kubelet
+can communicate with Docker on a CRI compatible endpoint.
+-->
+安装 Docker
+操作指南在 [Install Docker Engine - Enterprise on Windows Servers](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server#install-docker)。
```
Comment (javadoors) [inline content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:365]:
```suggestion
操作指南在[安装 Docker ](https://docs.microsoft.com/zh-CN/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server#install-docker)。
```

File: content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 3]
```diff
@@ -380,11 +340,77 @@ If you no longer have this command, or the token has expired, you can run `kubea
...
+
+[安装 cri-dockerd](https://github.com/Mirantis/cri-dockerd) 这是 kubelet 所必需的，
```
Comment (javadoors) [inline content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:367]:
```suggestion
[安装 cri-dockerd ](https://github.com/Mirantis/cri-dockerd) 这是 kubelet 所必需的，
```

File: content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 4]
```diff
@@ -380,11 +340,77 @@ If you no longer have this command, or the token has expired, you can run `kubea
...
 {{< note >}}
-If using **CRI-containerD** add `--cri-socket "npipe:////./pipe/containerd-containerd"` to the kubeadm call
+<!--
+Docker Engine does not implement the [CRI](/docs/concepts/architecture/cri/)
+which is a requirement for a container runtime to work with Kubernetes.
+For that reason, an additional service [cri-dockerd](/docs/concepts/architecture/cri/)
+has to be installed. cri-dockerd is a project based on the legacy built-in
+Docker Engine support that was [removed](/dockershim) from the kubelet in version 1.24.
+-->
+Docker 引擎没有实现 [CRI](/zh/docs/concepts/architecture/cri/)
```
Comment (javadoors) [inline content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:380]:
```suggestion
Docker 引擎没有实现 [CRI](/zh/docs/concepts/architecture/cri/)，
```

File: content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 5]
```diff
@@ -380,11 +340,77 @@ If you no longer have this command, or the token has expired, you can run `kubea
...
+这是容器运行时与 Kubernetes 一起工作的要求。
```
Comment (javadoors) [inline content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:381]:
```suggestion
CRI 是容器运行时与 Kubernetes 一起工作的要求。
```

**Pull request closed without merging**